From 2fcb94a04778708b13b6d36390000e97063460e6 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date: Fri, 31 May 2019 17:57:26 +0100
Subject: [PATCH] dts: Include CSI lane config for csi1

Without the include the peripheral is configured to have 0
data lanes, which doesn't allow much data to be passed.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>
---
 arch/arm/boot/dts/bcm2711-rpi-4-b.dts | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
+++ b/arch/arm/boot/dts/bcm2711-rpi-4-b.dts
@@ -1,6 +1,7 @@
 /dts-v1/;
 
 #include "bcm2711.dtsi"
+#include "bcm283x-rpi-csi1-2lane.dtsi"
 
 / {
 	compatible = "raspberrypi,4-model-b", "brcm,bcm2838", "brcm,bcm2837";
